koku/api/report/test/aws/test_views.py

```diff
@@ -600,3 +601,27 @@ def test_invalid_aws_category_key(self):
         url = reverse("reports-aws-costs") + "?group_by[aws_categroy:invalid]=value"
         response = self.client.get(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_units_consistency(self):
+        """Test if provider map used units match the report."""
+        file = "November-2023-None"
+        file_name = f"{file}.csv"
+        csv_units = set()
+        file_path = f"./koku/masu/test/data/aws/{file_name}"  # noqa: E501
+
+        provider_map_units = [
```

Comment:
So, I think it may be better to attempt to get these units directly from the provider map itself. Otherwise it just kind of feels like this test is just checking that the csv you created has these units in it?